src/cli/fixtures.ts

```diff
@@ -0,0 +1,151 @@
+import * as folio from 'folio';
+import type { LaunchOptions, BrowserContextOptions, Page } from '../../types/types';
+import type { PlaywrightTestArgs, PlaywrightTestOptions, PlaywrightWorkerArgs, PlaywrightWorkerOptions } from '../../types/test';
+
+export * from 'folio';
+export const test = folio.test.extend<PlaywrightTestArgs & PlaywrightTestOptions, PlaywrightWorkerArgs & PlaywrightWorkerOptions>({
+  browserName: [ 'chromium', { scope: 'worker' } ],
+  playwright: [ require('../inprocess'), { scope: 'worker' } ],
+  headless: [ undefined, { scope: 'worker' } ],
+  channel: [ undefined, { scope: 'worker' } ],
+  launchOptions: [ {}, { scope: 'worker' } ],
+
+  browser: [ async ({ playwright, browserName, headless, channel, launchOptions }, use) => {
+    if (!['chromium', 'firefox', 'webkit'].includes(browserName))
+      throw new Error(`Unexpected browserName "${browserName}", must be one of "chromium", "firefox" or "webkit"`);
+    const options: LaunchOptions = {
+      handleSIGINT: false,
+      ...launchOptions,
+    };
+    if (headless !== undefined)
+      options.headless = headless;
+    if (channel !== undefined)
+      options.channel = channel;
+    const browser = await playwright[browserName].launch(options);
+    await use(browser);
+    await browser.close();
+  }, { scope: 'worker' } ],
+
+  screenshot: 'off',
+  video: 'off',
+  acceptDownloads: undefined,
+  bypassCSP: undefined,
+  colorScheme: undefined,
+  deviceScaleFactor: undefined,
+  extraHTTPHeaders: undefined,
+  geolocation: undefined,
+  hasTouch: undefined,
+  httpCredentials: undefined,
+  ignoreHTTPSErrors: undefined,
+  isMobile: undefined,
+  javaScriptEnabled: undefined,
+  locale: undefined,
+  offline: undefined,
+  permissions: undefined,
+  proxy: undefined,
+  storageState: undefined,
+  timezoneId: undefined,
+  userAgent: undefined,
+  viewport: undefined,
+  contextOptions: {},
+
+  context: async ({ browserName, browser, screenshot, video, acceptDownloads, bypassCSP, colorScheme, deviceScaleFactor, extraHTTPHeaders, hasTouch, geolocation, httpCredentials, ignoreHTTPSErrors, isMobile, javaScriptEnabled, locale, offline, permissions, proxy, storageState, viewport, timezoneId, userAgent, contextOptions }, use, testInfo) => {
+    testInfo.snapshotSuffix = browserName + '-' + process.platform;
+    if (process.env.PWDEBUG)
+      testInfo.setTimeout(0);
+
+    const recordVideo = video === 'on' || video === 'retain-on-failure' ||
+      (video === 'retry-with-video' && !!testInfo.retry);
+    const options: BrowserContextOptions = {
+      recordVideo: recordVideo ? { dir: testInfo.outputPath('') } : undefined,
+      ...contextOptions,
+    };
+    if (acceptDownloads !== undefined)
+      options.acceptDownloads = acceptDownloads;
+    if (bypassCSP !== undefined)
+      options.bypassCSP = bypassCSP;
+    if (colorScheme !== undefined)
+      options.colorScheme = colorScheme;
+    if (deviceScaleFactor !== undefined)
```

Comment:
`reducedMotion` is missing, would be nice if we could have it more automated or at least leading to a TypeScript linting error when we forget a property for context/launch options.